src/commands/view/SelectComponent.js

```diff
@@ -100,7 +100,21 @@ module.exports = {
       this.onFrameScroll(e);
       this.updateAttached();
     }
+    var model = $(trg).data('model');
+    if (typeof model != 'undefined') {
 
+      if (!model.get("selectable")) {
+        var comp = model && model.parent();
```

Comment:
In this case, I think it makes more sense checking against `hoverable` property

Reply:
updated.